app/javascript/components/gtl/DataTable.jsx

```diff
@@ -135,19 +135,19 @@ export const DataTable = ({
         >
           {columns.map((column, columnKey) => (
             <td
+              // eslint-disable-next-line react/no-array-index-key
               key={`td_${columnKey}`}
               className={classNames({
                 narrow: row.cells[columnKey].is_checkbox || row.cells[columnKey].icon || row.cells[columnKey].is_button,
                 'is-checkbox-cell': row.cells[columnKey].is_checkbox,
               })}
             >
               { row.cells[columnKey].is_checkbox && !settings.hideSelect && !inEditMode()
-              && <label className="hiddenCheckboxLabel" id={`check_${row.id}`} aria-hidden="true">{`check_${row.id}`}</label>
+              && <label htmlFor="rowId" className="hiddenCheckboxLabel" id={`check_${row.id}`} aria-hidden="true">{`check_${row.id}`}</label>
```

Comment:
do you have ID=rowId for on row element or checkbox element?

Reply:
Changed the id and pushed changes.